appcenter-data/Data.d.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export const enum TimeToLive {
+    INFINITE = -1,
+    NO_CACHE = 0,
+    DEFAULT = -1
+}
+
+export const enum DefaultPartitions {
+    USER_DOCUMENTS = 'user',
+    APP_DOCUMENTS = 'readonly'
+}
+
+export class ReadOptions {
+    timeToLive: number | TimeToLive;
+    constructor(timeToLive: number | TimeToLive);
+}
+
+export class WriteOptions {
+    timeToLive: number | TimeToLive;
+    constructor(timeToLive: number | TimeToLive);
+}
+
+export interface DocumentWrapper {
+    deserializedValue: object | null;
+    eTag: string | null;
+    id: string;
+    isFromDeviceCache: boolean;
+    jsonValue: string | null;
+    lastUpdatedDate: Date | null;
+    partition: string | null;
+}
+
+export interface PaginatedDocuments {
+    currentPage: Page;
+    hasNextPage: () => Promise<boolean>;
+    getNextPage: () => Promise<Page>;
+    close: () => void;
+}
+
+export interface Page {
+    items: DocumentWrapper[];
```

Comment:
check if it can be null

Reply:
`items` can’t be null. I changed the behaviour of `getNextPage()` to reject if there are no additional pages.